tutorials/dataframe/df022_useKahan.C

```diff
@@ -73,7 +73,7 @@ public:
    }
 
    std::string GetActionName(){
-      return "THnHelper";
+      return "KahanSum";
```

Comment:
cling says it should be on 1 line: `std::string GetActionName() { return "KahanSum"; }`

Reply:
fixed